pisa/stages/discr_sys/pi_hyperplanes.py

```diff
@@ -131,41 +131,57 @@ def __init__(
     ):
         # -- Read fit_results_file and extract necessary info -- #
 
-        fit_results = from_file(fit_results_file)
+        if fit_results_file.endswith('.json'):
```

Comment:
We handle more than just files with ".json" suffixes (e.g. json.bz2, to start) . If csv is how you identify release files, check for csv, handle it, and your "else" condition should be to use `from_file`